test/test_components.py

```diff
@@ -15,6 +18,41 @@ def empty_network_5_buses():
     return network
 
 
+def test_geo_components_gpd_df(geo_components_network):
+    # Test whether 'geo_components' are 'geopandas.DataFrame' and conatin a 'geopandas.GeoSeries' in the 'geometry' column.
+    # geo_components = {"Bus", "Line", "Link", "Transformer"}
+
+    assert isinstance(geo_components_network.buses, geopandas.GeoDataFrame)
+    assert isinstance(geo_components_network.lines, geopandas.GeoDataFrame)
+    assert isinstance(geo_components_network.links, geopandas.GeoDataFrame)
+    assert isinstance(geo_components_network.transformers, geopandas.GeoDataFrame)
+
+    assert isinstance(geo_components_network.buses.geometry, geopandas.GeoSeries)
+    assert isinstance(geo_components_network.lines.geometry, geopandas.GeoSeries)
+    assert isinstance(geo_components_network.links.geometry, geopandas.GeoSeries)
+    assert isinstance(geo_components_network.transformers.geometry, geopandas.GeoSeries)
+
+
+def test_geo_component_add():
+    # Test whether 'geo_components' can be added to the network by add function.
+    # geo_components = {"Bus", "Line", "Link", "Transformer"}
+    # Point(0,1) and 'POINT (1 4)'(wkt_string) are valid and will be converted to shapely.geometry objects.
+    # {"None", "", np.nan, "nan"} are also valid values for geometry column while adding using add function and will be added as None.
+    # note - invalid strings will also be added as None.
```

Comment:
Use docstrings as applied in other cases

Reply:
Also consider the PEP convention: https://peps.python.org/pep-0008/#documentation-strings